src/database/models/notifications.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+module.exports = (Sequelize, DataTypes) => {
+  const Notifications = Sequelize.define('Notifications', {
+    modelId: { type: DataTypes.INTEGER, allowNull: false },
+    modelName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        isIn: {
+          args: [['Requests', 'Comments', 'Trips', 'Accommodations', 'AccommodationImages', 'Ratings', 'Likes']],
```

Comment:
For now, let's have only `Requests` and `Accommodations`. We can always add to this list if the need arises.